docs/source/example_use.rst

```diff
@@ -1,3 +1,343 @@
-Example Use Case
-================
+###############################
+Example Use Case: The News Room
+###############################
 
+Below we'll walk you through an example of how Orchestra could be used in a
+newsroom by journalists, editors, and photographers to craft a story. The code
+for this example can be found `in our github repo
+<https://github.com/unlimitedlabs/orchestra/tree/stable/journalism_workflow>`_.
+
+************
+The workflow
+************
+.. image:: ../static/img/journalism_workflow/workflow.jpg
+   :height: 300px
+
+The image above depicts our example workflow, which is composed of the
+following steps:
+
+* An editor finds a good story and sends a reporter off to investigate.
+
+* The reporter writes up a draft article.
+
+* A more experienced reporter then reviews the article and suggests
+  improvements.
+
+* In parallel with the reporting step, a photographer captures photos for the
+  story.
+
+* A senior photographer reviews the photos and selects the best ones.
+
+* The selected photos are resized and recolored for display across different
+  media.
+
+* Finally, a copy editor adds headlines and photo captions to complete the
+  story.
+
+To make things work in practice, there's also a hidden machine step at the
+beginning of the workflow to set up some google documents and folders for
+article writing and image storage.
+
+********************
+Running the workflow
+********************
+
+Setup
+=====
+
+If you haven't followed the :doc:`getting started guide <getting_started>` to
+set up Orchestra yet, you should do that now. Also, make sure that
+``'journalism_workflow'`` is in your ``INSTALLED_APPS`` setting.
+
+Additionally, the journalism workflow requires Google Apps integration to run,
```

Comment:
link to google apps integration website information where those things would be defined